### ISSUE TITLE
Missing isort rules in setup.cfg configuration

### ISSUE THREAD

**Issue by koutsenko** [wemake-services/wemake-python-styleguide#1934]
### What's wrong

You have an README https://github.com/wemake-services/wemake-python-styleguide/blob/master/README.md .

The setup.cfg configuration link points to https://wemake-python-stylegui.de/en/latest/pages/usage/configuration.html .
Only the [flake8] section described here, no [isort] configuration.
So, if you launch isort from CLI with default rules, e.g. for multiline imports, you'll get WPS warnings.

As result, it's too hard to fix WPS isort issues.

### How it should be

# https://github.com/wemake-services/wemake-python-styleguide/blob/master/setup.cfg#L68-L76
[isort]
include_trailing_comma = true
use_parentheses = true
multi_line_output = 3
line_length = 80

### Flake8 version and plugins

Latest

### pip information

Latest

### OS information

Win10x64

**Comment by orsinium** [comment]
AFAIK, the configuration docs mention the following:

> We also use flake8-isort to style our imports. You will need to update your configuration with the following lines: Otherwise, your isort will complain about your imports. Our [isort.toml](https://github.com/wemake-services/wemake-python-styleguide/blob/master/styles/isort.toml) file is available with the core settings for isort.

**Comment by sobolevn** [comment]
I agree that we should empathize more that there are other tools to configure.
Tools we use with their own configuration: `darglint`, `isort`
